game/scripts/npc/abilities/crystal_maiden_frostbite.txt

```diff
@@ -44,12 +44,12 @@
       "01"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "total_damage"                                    "150 200 250 300 350 450"
+        "total_damage"                                    "150 200 250 300 350 675"
```

Comment:
I think damage per second is always 100. So if you want to increase total damage you would need to increase the duration. But don't do that, and revert this.